samples/bluetooth/mesh/sensor_server/src/model_handler.c

```diff
@@ -473,6 +507,45 @@ static struct bt_mesh_sensor time_since_presence_detected = {
 	.get = time_since_presence_detected_get,
 };
 
+static int motion_sensed_get(struct bt_mesh_sensor_srv *srv, struct bt_mesh_sensor *sensor,
+			     struct bt_mesh_msg_ctx *ctx, struct bt_mesh_sensor_value *rsp)
+{
+	int err = bt_mesh_sensor_value_from_micro(sensor->type->channels[0].format,
+						  mot_sensed * 1000000LL, rsp);
+
+	if (err) {
+		printk("Error encoding motion sensed (%d)", err);
+	}
+	return err;
+};
+
+static struct bt_mesh_sensor motion_sensor = {
+	.type = &bt_mesh_sensor_motion_sensed,
+	.get = motion_sensed_get,
+	.settings = {
+		.list = (const struct bt_mesh_sensor_setting *)&presence_motion_threshold_setting,
```

Comment:
I'm note sure if you are supposed to refer to the same setting for two different sensors like this. Even if it's legal by the spec, I feel it's confusing, since if you set the threshold on the motion sensor, you wouldn't expect the presence sensor setting to change, and vice versa.

Reply:
Are we even required to have a setting here at all? Is this something required by the NLC Profile or something you added?

Reply:
I see your point. And no, the NLC profile does not require it. However, I felt like it was odd to have a "motion threshold" setting and not use this for the motion sensor.

Reply:
My point is that the setting is logically located under the Presence sensor. It's the `Presence Sensor/Motion Threshold` setting. If you want a setting here, you should define a separate setting, to avoid `Presence Sensor/Motion Threshold` changing unexpectedly when someone sets `Motion Sensor/Motion Threshold`. But in my opinion, since we already show how settings work for one of the sensors, that is enough, and we don't need anything here.